core/src/main/java/com/netflix/conductor/core/config/ConductorProperties.java

```diff
@@ -210,6 +210,9 @@ public class ConductorProperties {
     @DataSizeUnit(DataUnit.KILOBYTES)
     private DataSize maxWorkflowVariablesPayloadSizeThreshold = DataSize.ofKilobytes(256L);
 
+    /** Used to limit the size of task execution logs. */
+    private int taskExecLogSizeLimit = 1000;
```

Comment:
My thinking was to use an arbitrary large number as default and then figure out what is the reasonable number based on metrics, and then override it in our internal app. If we start here with a small number, we'll have to override this to a large number internally in the initial step

Reply:
Yes. I think it's better to override to a larger number internally because this number will be part of the public release. If we set it to 1000 and change back to a lesser number based on metrics. From an OSS release perspective, it's a backwards incompatible change.

I see a couple of options,

1. We can set this number as 1000 and not change it in a future release. But 1000 is such a big number it's almost like not having a limit.
2. We can set it to a smaller number in the OSS release, which gives the possibility to change it in the future in a backwards compatible way.

Reply:
I'll change it to 10 and override it internally